src/renderer.h

```diff
@@ -93,6 +94,14 @@ enum RendererTextureType
     YUV
 };
 
+enum ViewId

```

Comment:
This is not even supposed to exist, because of the aforementioned reasons.

Reply:
Please see comment above. If you decide its best to revert to using backendViewId then I will delete this enum.

Reply:
As explained in the previous comment, I reverted to using the `backendViewId` variable so I deleted this enum.